source/reference/minio-mc/mc-idp-ldap-accesskey-create.rst

```diff
@@ -0,0 +1,214 @@
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc idp ldap accesskey create minio/
+
+   .. tab-item:: SYNTAX
+
+      The command has the following syntax:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc [GLOBALFLAGS] idp ldap accesskey create                   \
+                                          ALIAS                       \
+                                          [--access-key <value>]      \
+                                          [--secret-key <value>]      \
+                                          [--policy <value>]          \
+                                          [--name <value>]            \
+                                          [--description <value>]     \
+                                          [--expiry-duration <value>] \ 
+                                          [--login <site>]
+
+      - Replace ``ALIAS`` with the :ref:`alias <alias>` of a MinIO deployment configured for AD/LDAP integration.
+
+      .. include:: /includes/common-minio-mc.rst
+         :start-after: start-minio-syntax
+         :end-before: end-minio-syntax
+
+
+Parameters
+~~~~~~~~~~
+
+.. mc-cmd:: ALIAS
+   :required:
+
+   The :ref:`alias <alias>` of the MinIO deployment configured for AD/LDAP.
+
+   For example:
+
+   .. code-block:: none
+
+         mc idp ldap accesskey create minio
+
+.. mc-cmd:: --access-key
+   :optional:
+
+   An access key to use for the account.
+
+   Requires :mc-cmd:`~mc idp ldap accesskey create --secret-key`
+
+.. mc-cmd:: --secret-key
+   :optional:
+
+   A secret to use for the account.
+
+   Requires :mc-cmd:`~mc idp ldap accesskey create --access-key`
+
+.. mc-cmd:: --policy
+   :optional:
+
+   File path to the JSON-formatted policy to use for the account.
+
+   If not specified, the account uses the same policy as the authenticated user.
+
+.. mc-cmd:: --name
+   :optional:
+
+   A human-readable name to use for the account.
+
+.. mc-cmd:: --description
+   :optional:
+
+   Add a description for the service account.
+   For example, you might specify the reason the access key exists.
+
+.. mc-cmd:: --expiry-duration
+   :optional:
+
+   Length of time the access key pair should remain valid for use in ``#d#h#s`` format.
+       
+   For example, ``7d``, ``24h``, ``5d12h30s`` are valid strings.
+
+   Mutually exclusive with :mc-cmd:`~mc idp ldap accesskey create --expiry`.
+
+.. mc-cmd:: --expiry
+   :optional:
+
+   The date after which the access key expires.
+   Enter the date in YYYY-MM-DD format.
+
+   For example, to expire the credentials after December 31, 2024, enter ``2024-12-31``.
+
+   Mutually exclusive with :mc-cmd:`~mc idp ldap accesskey create --expiry-duration`.
+
+.. mc-cmd:: --login
+   :optional:
+
+   Prompts the user to log in using the LDAP credentials to use to generate the access key.
+   Specify the URL of the LDAP-configured MinIO Server to use for the login prompt.
+
+   Requires an interactive terminal.
+
+Examples
```

Comment:
Should be an H2 right?

Reply:
Suggest moving this after Global Flags and promotion

Reply:
Corrected heading levels and moved.